site/content/en/docs/Reference/fleet.md

```diff
@@ -58,9 +58,11 @@ spec:
         initialDelaySeconds: 30
         periodSeconds: 60
 {{% feature publishVersion="1.1.0" %}}
```

Comment:
This is going to make an ugly gap in the final rendering - have a look at #1081 - if you review and merge that first, you can take advantage of the split I've already got in there.

Reply:
I'll merge your PR and rebase.